build.gradle

```diff
@@ -372,7 +372,7 @@ task integTestRemote(type: RestIntegTestTask) {
     }
 }
 
-String bwcVersion = "1.1.0.0"
+String bwcVersion = "2.4.0.0"
```

Comment:
nit: We can maybe set this at the top of constants and also use it for fetching correct version in line 41/43. just to minimize number of place we are changing version number on each increment in main?

Reply:
> nit: We can maybe set this at the top of constants and also use it for fetching correct version in line 41/43. just to minimize number of place we are changing version number on each increment in main?

moved bwc related constants to the build script block